app/src/main/java/com/mghelas/internship_playground/sweetsfactory/Chocolate.java

```diff
@@ -0,0 +1,29 @@
+package com.mghelas.internship_playground.sweetsfactory;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+public class Chocolate extends Sweet {
```

Comment:
Why do have only Chocolate? You have the factory of sweets)) You can add something else

Reply:
i asked "why?" Can you prove it? or explain me

Reply:
i copy pasted it